components/update_client/net/network_impl_cobalt.cc

```diff
@@ -184,25 +185,26 @@ void NetworkFetcherCobaltImpl::OnURLFetchResponseStarted(
 void NetworkFetcherCobaltImpl::OnURLFetchComplete(
     const net::URLFetcher* source) {
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
+  LOG(INFO) << "cobalt::updater::NetworkFetcher::OnURLFetchComplete";
```

Comment:
nit: remove

Reply:
Hmm, this line is not for my debugging purpose. And it's basically a copy past of this file: cobalt/updater/network_fetcher.cc. In addition, if you look at the functions in this file, each function has a LOG(INFO), so I think we should keep it here.

Reply:
Got it, assumed it was for debugging